### PR TITLE
Vent hum is now sound played by vents

### PR DESCRIPTION
**Pull request by comma**
Removed it from area ambience
Made turfs raise opacity change event when their opacity counter is updated (otherwise sound players never reacted to doors opening)
Implements https://github.com/NebulaSS13/NebulaFeatureRequests/issues/17

### CODE REVIEW

File: code/modules/atmospherics/components/unary/vent_pump.dm [hunk 1]
```diff
@@ -554,6 +557,19 @@
...
+/obj/machinery/atmospherics/unary/vent_pump
+	
+/obj/machinery/atmospherics/unary/vent_pump/power_change()
+	. = ..()
+	if(.)
+		update_sound()
+
+/obj/machinery/atmospherics/unary/vent_pump/proc/update_sound()
+	if(can_pump())
+		sound_token = GLOB.sound_player.PlayLoopingSound(src, "[type]_[x]_[y]_[z]", 'sound/machines/vent_hum.ogg', 15, range = 14, falloff = 4)
```
Comment (PsiOmegaDelta) [inline code/modules/atmospherics/components/unary/vent_pump.dm]: Due to BYOND-limitations we can only have 1024 unique sound-id sources and vents would take up a big part of those
Generally objects playing the same sound should all utilize the same sound id

Comment (comma) [reply code/modules/atmospherics/components/unary/vent_pump.dm]: Changed it to only enforce uniqueness inside zlevel. As far as people can hear, they're all unique.

File: code/modules/atmospherics/components/unary/vent_pump.dm [hunk 2]
```diff
@@ -554,6 +558,21 @@
 	. = ..()
 	air_contents.volume = ATMOS_DEFAULT_VOLUME_PUMP + 500 //meant to match air injector
 
+/obj/machinery/atmospherics/unary/vent_pump
```
Comment (MistakeNot4892) [inline code/modules/atmospherics/components/unary/vent_pump.dm]: Random path?

Comment (comma) [reply code/modules/atmospherics/components/unary/vent_pump.dm]: fixed

**Review by afterthought2 — APPROVED** [review]
Not wild about the implementation, but don't have a reasonable alternative suggestion. The turf opacity thing is also a bit screwy; may be worth adding a comment about it in the opacity set file.

**Pull request merged**